### PR TITLE
Setup/cypress

### LINKED ISSUE #14: Main Page Testing
- [ ] Create tests for Main page using Cypress

### LINKED ISSUE #15: Single Brewery Page Testing
- [ ] Create tests for single brewery page using Cypress

### PR DESCRIPTION
**Pull request by rachellhill**
# Description
Please include a summary of the changes and/or which issue is fixed. Please also include relevant motivation and context. 

- Adds Cypress testing for the home page and brewery details page 

Were there any issues that arose? 

Is there anything you need from your teammates? 

Any other comments, questions, or concerns?

## Issues
Closes #15 
Closes #14 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?
- [x] React Dev Tools
- [x] Functionality check through web browser
- [x] Ran Cypress test suite

# Checklist:
- [x] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..).
- [x] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have deleted all comments after functionality and understanding is final.
- [x] I have added any issues closed by this PR in the Issues section.
- [x] I have added others as reviewers.

### CODE REVIEW

File: cypress/e2e/1-getting-started/todo.cy.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
+
+// Welcome to Cypress!
+//
+// This spec file contains a variety of sample tests
+// for a todo list app that are designed to demonstrate
```
Comment (aominhlong) [inline cypress/e2e/1-getting-started/todo.cy.js:6]: We can remove the informational comments when we refactor

**Review by stephanie-roe — APPROVED** [review]
This is awesome! I like how we stubbed the requests in the before each and even tested for Router

File: cypress/e2e/home_spec.cy.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// describe('home_spec.cy.js', () => {
+//   it('should visit', () => {
+//     cy.visit('/')
+//   })
+// })
+
```
Comment (userigorgithub) [inline cypress/e2e/home_spec.cy.js:6]: Nice testing for every element on a page!

File: cypress/e2e/home_spec.cy.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// describe('home_spec.cy.js', () => {
+//   it('should visit', () => {
+//     cy.visit('/')
```
Comment (aominhlong) [inline cypress/e2e/home_spec.cy.js:3]: We can remove this comment

**Comment by aominhlong** [comment]
Tests look good so far. We can get rid of the todo test file and unnecessary comments! Good work!

File: cypress/e2e/brewery_details_spec.cy.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+describe('Brewery Details', () => {
```
Comment (userigorgithub) [inline cypress/e2e/brewery_details_spec.cy.js:1]: GJ testing everything so far, it seems.

**Review by userigorgithub — APPROVED** [review]
Great job with implementing Cypress testing! We will have to add more tests as we go though.

**Pull request merged**